### ISSUE TITLE
suffix flag has been removed

### ISSUE THREAD

**Issue by angrz** [gojuno/minimock#23]
`suffix      = flag.String("s", "_mock_test.go", "output file name suffix which is added to file names when multiple interfaces are given")`
Hi, there was a suffix flag before switch to gowrap, it wasn't  marked as deprecated. I would be very grateful if you could bring that flag back.

**Comment by beono** [comment]
Yep. Please, bring it back. I don't need `_mock_test.go` suffix.

**Comment by beono** [comment]
@hexdigest 

Well, that makes sense, however if I follow your suggestion then I will have too many files in my packages.

Imagine I have a file that contains 10 interfaces.
According to your idea I will get something like:
```
foo.go
foo_test.go
inteface1_mock_test.go
inteface2_mock_test.go
inteface3_mock_test.go
...
inteface10_mock_test.go

```

Therefore I would like to have a subpackage `mocks` which will hide all "useless" files.
I will keep foo_test.go in the same directory as foo.go since I want to be able to test unexported things.
The fact that the compiles will have to compile my mocks I consider as insignificant at the moment in my projects.

The option `-o` forces me to define the full name of the mock file, though it obviously can be taken from the interface definition.

So if your motivation is to force developers to use package structure, I think it is achieved by the fact minimock does by default.
If you have other reasons (such as complexity of the minimock code base), then it's up to you for sure.

**Comment by hexdigest** [comment]
@angrz @beono thanks for your comments.
This issue is fixed in 2.1.0, please update.
